frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableColumnList/TableColumn/TableColumn.tsx

```diff
@@ -12,16 +12,38 @@ type TableColumnProps = {
   handleId: string
   isSource: boolean
   targetCardinality?: CardinalityType | undefined
+  isHovered?: boolean
+  isSelectedTable?: boolean
+  releatedColumns: string[]
 }
 
 export const TableColumn: FC<TableColumnProps> = ({
   column,
   handleId,
   isSource,
   targetCardinality,
+  isHovered,
+  isSelectedTable,
+  releatedColumns,
 }) => {
+  const getTableColumnBackgroundColor = () => {
+    if (isSelectedTable && releatedColumns.includes(column.name)) {
+      return '#22392f'
+    }
+    if (isHovered) {
+      return '#434546'
+    }
+    return undefined
+  }
```

Comment:
I would like it to be written in CSS Class as much as possible and implemented in JavaScript by reattaching classes.
Also please use CSS Variables.